rcl/test/rcl/test_arguments.cpp

```diff
@@ -698,6 +707,35 @@ TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_bad_remove_ros_
   EXPECT_EQ(0, nonros_argc);
 }
 
+TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_bad_alloc_remove_ros_args) {
+  const char * const argv[] = {
+    "process_name", "-d", "--ros-args", "-r", "__ns:=/foo/bar", "-r", "__ns:=/fiz/buz", "--",
+    "--foo=bar", "--baz", "--ros-args", "--ros-args", "-p", "bar:=baz", "--", "--", "arg",
+  };
+  const int argc = sizeof(argv) / sizeof(const char *);
+
+  rcl_allocator_t alloc = rcl_get_default_allocator();
+  rcl_arguments_t parsed_args = rcl_get_zero_initialized_arguments();
+  rcl_ret_t ret = rcl_parse_arguments(argc, argv, alloc, &parsed_args);
+  ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+  OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT(
+  {
+    EXPECT_EQ(RCL_RET_OK, rcl_arguments_fini(&parsed_args));
+  });
+
+  int nonros_argc = 0;
+  const char ** nonros_argv = NULL;
+  rcl_allocator_t bomb_alloc = get_time_bombed_allocator();
```

Comment:
Where is `get_time_bombed_allocator` coming from? It's not spelled correctly, but secondly I don't see an appropriate include.

Reply:
I added that allocator with #743. As it also tested `arguments.c` it's already included in `test_arguments.cpp'.

Reply:
Oh, I didn't look at allocator_testing_utils.h close enough. Thanks for the info! But it should be `get_time_bomb_allocator`, "time_bombed" doesn't make a ton of sense. Feel free to fix in a follow-up PR.